CMakeLists.txt

```diff
@@ -65,7 +65,7 @@ if(MSVC)
   # Disabled bogus warnings for CPU intrinsics:
   # C4310: cast truncates constant value
   # C4324: 'marian::cpu::int16::`anonymous-namespace'::ScatterPut': structure was padded due to alignment specifier
-  set(DISABLE_GLOBALLY "/wd\"4310\" /wd\"4324\"")
+  set(DISABLE_GLOBALLY "/wd\"4310\" /wd\"4324\" /wd\"4702\"")
```

Comment:
Please add a comment what kind of warning this is.

Reply:
Comment added. I tried disabling this only in the Debug build, but bogus warnings show up for the Release build too so I keep it disabled globally (as it is in the VS project file).